src/internal/connector/onedrive/drive.go

```diff
@@ -116,21 +67,11 @@ func siteDrives(ctx context.Context, service graph.Servicer, site string) ([]mod
 
 func userDrives(ctx context.Context, service graph.Servicer, user string) ([]models.Driveable, error) {
 	var (
-		hasDrive        bool
 		numberOfRetries = 3
 		r               models.DriveCollectionResponseable
+		err             error
 	)
 
-	hasDrive, err := hasDriveLicense(ctx, service, user)
-	if err != nil {
-		return nil, errors.Wrap(err, user)
-	}
-
-	if !hasDrive {
-		logger.Ctx(ctx).Debugf("User %s does not have a license for OneDrive", user)
-		return make([]models.Driveable, 0), nil // no license
-	}
```

Comment:
The check below is sufficient.

Reply:
@vkamra How does this work if the user drive is not provisioned yet?

Reply:
The check on line 80 below catches that case.